test/experimental/main_test.go

```diff
@@ -24,7 +24,7 @@ import (
 	"testing"
```

Comment:
This file needs a build flag set on it, maybe `+build e2e,experimental`

Reply:
Do we need to have another flag, given we already have `e2e`? Isn't that enough? I see we don't make difference between e2e/conformance/rekt in terms of build flags, they all use `e2e`